server.cpp

```diff
@@ -197,16 +199,20 @@ void handle_request(struct server_app *app, int client_socket) {
     if (requested_path == "/") {
         requested_path = "/index.html";
     }
-
+    //std::cout<<"requested_path";
+    //std::cout<<requested_path;
     // TODO: Implement proxy and call the function under condition
     // specified in the spec
-    // if (need_proxy(...)) {
-    //    proxy_remote_file(app, client_socket, file_name);
-    // } else {
-    serve_local_file(client_socket, requested_path);
-    //}
+    if (requested_path.length() >= 3) {
+        if (requested_path.substr(requested_path.length() - 3, 3) == ".ts") {
```

Comment:
can make this one if statement using &&
Could also consider using the determine_filetype function, adding a new `ft_ts` enum